### DISCUSSION TITLE
Resume Training Question

### DISCUSSION THREAD

**Discussion by caniyabanci76** [bmaltais/kohya_ss#1415] [Q&A]
I understand what `--save_state` does and what `--resume` do but what I'd like do know is:

when launching a run to resume, is the `--pretrained_model_name_or_path` parameter ignored? or should it point to the ORIGINAL model that the training started on? Or should it point to the model that i wish to resume FROM?

**Comment by TimothyAlexisVass** [comment]
## Put the original model as pretrained model:
### `--pretrained_model_name_or_path="path/to/original/model"`
## Put the file optimizer generated by `--save_state` in the resume command:
### `--resume="path/to/optimizer/state"`

**Comment by BrentWilkins** [comment]
If you want to use the GUI that this repo provides, I think it's as simple as using the following field. Am I wrong? Seems to work for me.
![Screenshot from 2023-11-01 20-06-37](https://github.com/bmaltais/kohya_ss/assets/218822/741063cc-32d3-45e3-9246-cd644260f4d8)
